iogt/templatetags/tracking_tags.py

```diff
@@ -0,0 +1,19 @@
+from django import template
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+register = template.Library()
+
+
+@register.inclusion_tag('tracking_matomo_js_disabled_code.html')
+def tracking_matomo_js_disabled_code():
+    try:
+        id = settings.MATOMO_SITE_ID
+    except AttributeError:
+        raise ImproperlyConfigured('MATOMO_SITE_ID does not exist.')
```

Comment:
I know this is what the 'django-matomo' app does, but it's not useful or helpful for the site to blow up just because Matomo is not set up correctly - it is entirely optional. If we want to do this kind of thing, we should do it during startup, and take into account whether Matomo tracking is enabled or not.